subsys/bluetooth/host/testing.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file testing.h
+ * @brief Internal API for Bluetooth testing.
+ */
+
+/*
+ * Copyright (c) 2017 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
No header guard?

Reply:
I didn't saw include guard in any of the bluetooth's non-public headers

Reply:
Didn't see the RFC tag ...